tests/lib/ringbuffer/src/main.c

```diff
@@ -21,6 +21,17 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(test);
 
+/* Max size is used internally in algorithm. Value is decreased in the test to
```

Comment:
Max size is used internally in the algorithm. Value is decreased in the test to trigger the rewind algorithm.